addons/mrp/static/src/components/bom_overview_control_panel/mrp_bom_overview_control_panel.js

```diff
@@ -17,7 +18,7 @@ export class BomOverviewControlPanel extends Component {
     //---- Handlers ----
 
     updateQuantity(ev) {
-        const newVal = isNaN(ev.target.value) ? 1 : parseInt(ev.target.value);
+        const newVal = isNaN(ev.target.value) ? 1 : parseFloat(formatFloat(1, parseFloat(ev.target.value), { digits: [false, this.precision] }));
```

Comment:
```suggestion
        const newVal = isNaN(ev.target.value) ? 1 : parseFloat(formatFloat(parseFloat(ev.target.value), { digits: [false, this.precision] }));
```
This doesn't work as `formatFloat(value, options = {})` only take two arguments.
Also kinda feels a bit overkill just to eliminate unwanted decimals, but I guess we can't do much about it? :thinking:

Reply:
Sorry about that, I pushed the wrong version of my code.
Otherwise, yes, we don't have another cleaner choice to eliminate unwanted decimals, we can use this: Number(parseFloat(ev.target.value).toFixed(precision))
but that's less clean.

Reply:
Yes, there's no need to change the decimal accuracy for this bug because the issue comes from the call to `parseInt`. However, I also wanted to show that if we add more decimals, they should be properly cleaned up.
I updated my commit message.

Reply:
@clesgow Thanks for the review ;)